elkm1_lib/areas.py

```diff
@@ -83,16 +91,16 @@ def _as_handler(self, armed_statuses, arm_up_states, alarm_states):
             self.elk.send(az_encode())
 
     def _ee_handler(
-        self, area, is_exit, timer1, timer2, armed_status
-    ):  # pylint: disable=too-many-arguments
-        area = self.elements[area]
+        self, msg_area: int, is_exit: bool, timer1: int, timer2: int, armed_status: str
+    ) -> None:
+        area: Area = cast(Area, self.elements[msg_area])
```

Comment:
```suggestion
        area = cast(Area, self.elements[msg_area])
```

The `: Area` is redundant with the cast